### PR TITLE
[13.0][ADD] sale_coupon_order_pending: New module

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT41520

Please review @sergio-teruel @CarlosRoca13

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:

File: sale_coupon_order_pending/views/coupon_coupon_views.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                                <t t-if="record.can_be_applied_to_order.raw_value">
+                                    <button
+                                        name="action_apply_partner_coupon"
+                                        string="Apply"
+                                        class="btn btn-primary"
+                                        type="object"
+                                    >Apply</button>
+                                </t>
+                                <t t-else="">
+                                    <div class="alert alert-info mb-0"><i
```
Comment (stefan-tecnativa) [inline sale_coupon_order_pending/views/coupon_coupon_views.xml]:
```suggestion
                                    <div class="alert alert-info mb-0" role="alert"><i
```

When installing the module, it's complaining that an alert class must have an alert, alertdialog or status role.

**Review by stefan-tecnativa — APPROVED** [review]
Functionally tested and working well 👍️

File: sale_coupon_order_pending/views/coupon_coupon_views.xml (hunk 1 above)
Comment (chienandalu) [reply sale_coupon_order_pending/views/coupon_coupon_views.xml]: Done. Thanks :)

**Comment by chienandalu** [comment]
This can be merged IMO

**Pull request merged**